doc/quick_start.rst

```diff
@@ -10,10 +10,60 @@ Estimate your prediction intervals
 1. Download and install the module
 ----------------------------------
 
+Install via `pip`:
 
-2. Run PredictionInterval
--------------------------
+.. code:: python
+
+    pip install mapie
+
+To install directly from the github repository :
+
+.. code:: python
+
+    pip install git+https://github.com/simai-ml/MAPIE
+
+
+2. Run MapieRegressor
+---------------------
+
+Before calling MAPIE, we first define a sklearn-compatible regressor as well as training and test sets.
+MAPIE is compliant with the standard scikit-learn API.
```

Comment:
Il manque une phrase du type "let us start with a basic regression problem". La phrase "MAPIE is compliant with the standard scikit-learn API." devrait être placée après ce bout de code. De plus, il y a une redondance partielle avec la phrase suivante : "Like any scikit-learn regressor, MAPIE follows the following sequential ``fit`` and ``predict`` process. "

Reply:
J'ai modifié les phrases pour enlever la répétition.